templates/SilverStripe/Forms/GridField/GridFieldPaginator_Row.ss

```diff
@@ -2,16 +2,22 @@
 	<td class="grid-field__paginator bottom-all" colspan="$Colspan">
 		<% if $OnlyOnePage %>
 		<% else %>
-			<div class="grid-field__paginator__controls datagrid-pagination">
+			<nav aria-label="<%t SilverStripe\\Forms\\GridField\\GridFieldPaginator.Pagination 'Pagination' %>"
```

Comment:
This label is too generic, as multiple gridfields on the same page will end up having the same label.

See https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/navigation.html
> If a page includes more than one navigation landmark, each should have a unique label.

See https://design-system.w3.org/components/pagination.html

> Where there are multiple `<nav>` elements on a single page, they should all have a unique accessible name.